app/components/domains-table.tsx

```diff
@@ -89,6 +90,25 @@ export default function DomainsTable(props: DomainsTableProps) {
     }
   }
 
+  function renderDnsRecordName(domainName: string) {
+    const words: string[] = domainName.split('.');
```

Comment:
Don't bother including types when they can be inferred.  Here, the value will automatically be `:string[]` due to the assignment.